Console/Command/QueueShell.php

```diff
@@ -11,7 +11,9 @@
 
 App::uses('Folder', 'Utility');
 App::uses('File', 'Utility');
-App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
+if (!class_exists('HipChat\HipChat')) {
```

Comment:
Tasks are intended to be re-usable between shells... I don't think anything else actually uses it, but if it does get re-used it would fail in that case.

I went over some other options with Simon and Derek and this seems like the least painful.  We need to upgrade to Hipchat API v2 at some point in the future so this could change at that time.

> On Nov 12, 2015, at 5:27 PM, Joseph Fung <notifications@github.com> wrote:
> 
> In Console/Command/QueueShell.php:
> 
> > @@ -11,7 +11,9 @@
> >  
> >  App::uses('Folder', 'Utility');
> >  App::uses('File', 'Utility');
> > -App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
> > +if (!class_exists('HipChat\HipChat')) {
> Then if it's already loaded, wouldn't the fix to be to stop including it in the Kudos task?
> 
> —
> Reply to this email directly or view it on GitHub.
>